src/cmd/deploy.ts

```diff
@@ -775,7 +775,6 @@ export default class DefenderDeploy {
             notificationChannels: match.notifyConfig?.notifications.map(
               (n: DefenderNotificationReference) => n.notificationId,
             ),
-            severityLevel: match.notifyConfig?.severityLevel,
```

Comment:
@shahnami Had a type issue here does it make sense to remove it?

Reply:
No, you'll need to yarn link the latest SDK main branch since the SDK hasn't been published yet.